packages/devextreme/js/localization/messages/ar.json

```
@@ -288,6 +288,10 @@
         "dxScheduler-noTimezoneTitle": "بلا منطقة زمنية",
         "dxScheduler-moreAppointments": "{0} المزيد",
 
+        "dxCalendar-currentDay": "Today",
+        "dxCalendar-currentMonth": "Current month",
+        "dxCalendar-currentYear": "Current year",
+        "dxCalendar-currentYearsRange": "Current years range",
```

```suggestion
        "dxCalendar-currentYearsRange": "Current year range",
```

Attributive nouns ("year" in this case) cannot be in plural form even if we talk about multiple objects. Please change it to a singular "year" in every localization file and in your code.